### ISSUE TITLE
Assertion fail when using an ui-dropdown

### ISSUE THREAD

**Issue by clawfire** [Semantic-Org/Semantic-UI-Ember#51]
When I use the `{{ui-dropdown}}` component in my template, I got this error :
```
Uncaught Error: Assertion Failed: You must use Ember.set() to set the `childViews` property (of <bml-admin@component:ui-dropdown::ember1047>) to ``.
```

Ember 1.13.5 & Ember-data 1.13.5

**Comment by nathanpalmer** [comment]
Hello @clawfire. Are you using the `master` branch or `prep-1.0`?

**Comment by clawfire** [comment]
`master`

**Comment by nathanpalmer** [comment]
We just released version 0.9.0 (which has some breaking changes in it.) Let me know if you're still having issues there. There were significant changes in this branch in order for us to move away from the Ember views that have been deprecated.

**Comment by aaronbhansen** [comment]
Closing no activity
